### PR TITLE
feat(geoUtils): add greek tax offices data and functions

### PR DESCRIPTION
**Pull request by DionisisChris**

### CODE REVIEW

**Comment by DionisisChris** [comment]
More tests need to be added, maybe more functions and a general refinement.(Will also add a description in the pr)

File: src/geoUtils.ts [hunk 1]
```diff
@@ -205,3 +211,101 @@ export function findByPostalCode(
...
+  normalized = normalized.replace(/[ \-_!@#$%^&*()]/g, ""); // Remove spaces and special characters
+  return normalized.toLocaleUpperCase(); // Convert to uppercase
+}
+
+//Compare greek words
+export function compareGreekStrings(stringA: string, stringB: string): boolean {
+  if (normalizeString(stringA) === normalizeString(stringB)) {
+    return true;
+  }
+  return false;
```
Comment (tsevdos) [inline src/geoUtils.ts:310]: You can simply return  replace thebelow 4 lines with:

```suggestion
  return normalizeString(stringA) === normalizeString(stringB));
```

**Comment by DionisisChris** [comment]
added the postal code function and more tests.

**Review by tsevdos — APPROVED** [review]

**Comment by tsevdos** [comment]
I couldn't manage to merge this branch (something went wrong with the main branch), but I created and successfully merged the  `feat/tax-offices` branch (https://github.com/tsevdos/elUtils/pull/76)

**Pull request closed without merging**